students/srepking/Lesson2/FizzBuzz.py

```diff
@@ -0,0 +1,11 @@
+def fizzbuzz():
+    for i in range(1,101):
+        if i%3==False and i%5==False:
+            print('FizzBuzz')
```

Comment:
when comparing to False and None you should use `is` to ensure it compares as an object:

```
>>> 3 % 3
0
>>> 3 % 3 is False
False
>>> 3 % 3 == False
True
>>>
```
so it should be:
`if i % 3 == 0`